doc/architecture-design.md

```diff
@@ -29,7 +29,7 @@
 - [**Script**](#script)
 - [**Upstream**](#upstream)
 - [**Router**](#router)
-- [**Consumer**](#consumer)
+- [**Consumer**](#consumer1)
```

Comment:
If you change this to `- [**Consumer**](#consumer-1)`, it will be correctly linked to the line `497` (https://github.com/apache/apisix/blame/master/doc/architecture-design.md#L497)

Reply:
only update here should be enough @96RadhikaJadhav